test/OpenTelemetry.Exporter.OpenTelemetryProtocol.Tests/OtlpTraceExporterTests.cs

```diff
@@ -370,7 +370,11 @@ public void ToOtlpSpanTest()
             links: childLinks);
 
         Assert.NotNull(childActivity);
+#pragma warning disable
+        // TODO: Remove this when SetStatus is deprecated
         childActivity.SetStatus(Status.Error);
+#pragma warning disable
+        childActivity.SetStatus(ActivityStatusCode.Error);
```

Comment:
I don't see where the status code is asserted in this test.
I see status description on line 397.

Just curious, if you call `SetStatus` and provide two different codes, which would win first or last?

Reply:
There is a separate test for this:https://github.com/open-telemetry/opentelemetry-dotnet/blob/940d5b528dcfe532795f088a968b7cb13695f550/test/OpenTelemetry.Exporter.OpenTelemetryProtocol.Tests/OtlpTraceExporterTests.cs#L541